browser/pages/confirm/controller.js

```diff
@@ -35,14 +35,10 @@ class ConfirmController {
     // IF the JDK is not Configured then you can't install devstudio
     $scope.$watch('checkboxModel.cdk.selectedOption', function watchCdkSelectionChange(nVal) {
       if(nVal=='install') {
-        if($scope.checkboxModel.vagrant.selectedOption == 'detected'
-          && !$scope.checkboxModel.vagrant.hasOption('detected')) {
-          $scope.checkboxModel.vagrant.selectedOption = 'install';
-        }
-        if($scope.checkboxModel.virtualbox.selectedOption == 'detected'
-          && !$scope.checkboxModel.virtualbox.hasOption('detected')) {
-          $scope.checkboxModel.virtualbox.selectedOption = 'install';
-        }
+        // if($scope.checkboxModel.virtualbox.selectedOption == 'detected'
+        //   && !$scope.checkboxModel.virtualbox.hasOption('detected')) {
+        //   $scope.checkboxModel.virtualbox.selectedOption = 'install';
+        // }
```

Comment:
Thanks for the clarification. However, it creates a weird situation on mac, see the screenshot:
![screen shot 2017-01-10 at 16 10 52](https://cloud.githubusercontent.com/assets/4181232/21811256/54fe80a4-d74f-11e6-8024-9bde25688eef.png)
It now allows to deselect virtualbox but doesn't let you proceed if you do.